<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DADpastClass</class>
 <widget class="QMainWindow" name="DADpastClass">
  <property name="geometry">
   <rect>
    <x>0</x>  
    <y>0</y>
    <width>584</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DADpast</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>draw</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>ok</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>blend</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>584</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>file</string>
    </property>
    <addaction name="actionInput_src"/>
    <addaction name="actionInput_target"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionInput_src">
   <property name="text">
    <string>input src</string>
   </property>
  </action>
  <action name="actionInput_target">
   <property name="text">
    <string>input target</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="dadpast.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>DADpastClass</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>DADpastClass</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>447</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
